File: 1009.cpp
#include <iostream>
#include <iomanip>>

using namespace std;

int main() {
 
 string nome;
 double salario, vendas;
 
 cin >> nome >> salario >> vendas;
 double total = salario + vendas * 0.15;
 cout << "TOTAL = R$ " << fixed << setprecision(2) << total << endl;
 
    return 0;
}

File: 1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(text) {
    return lines.shift();
}
var nome = prompt();
var salarioFixo = parseFloat(prompt());
var totaldeVendas = parseFloat(prompt());
var comissao = totaldeVendas * 0.15;
var salario = salarioFixo + comissao;
console.log("TOTAL = R$ " + salario.toFixed(2));